src/AppBundle/Resources/views/Report/Contact/summary.html.twig

```diff
@@ -21,6 +21,9 @@
          {{ 'summaryPage.weAskAbout' | trans(transOptions) }}
     </p>
 
+    {# @biggs remove me #}
+    <a href="{{ section_link_params(report, 'contacts', +1).link }}">{{ 'prevNextLinks.sections.contacts' | trans({}, 'report-common') }}</a>
```

Comment:
@Biggsen example of usage to get link to next section. use "-1" for prev section